paddle/framework/operator.h

```diff
@@ -413,7 +561,8 @@ class OperatorWithKernel : public OperatorBase {
   }
 
  protected:
-  virtual void InferShape(const InferShapeContext& ctx) const = 0;
+  virtual void InferShape(const InferShapeContext& ctx) const {}
+  virtual void InferShape(const InferShapeContextBase& ctx) const {}
```

Comment:
Why there two interfaces?

Reply:
The old one is for compiling and test, has been removed after transform all the old operators.